src/Google.Cloud.Diagnostics.Debug/Google.Cloud.Diagnostics.Debug/LoggingClient.cs

```diff
@@ -130,5 +129,33 @@ private string SubstituteLogMessageFormat(string messageFormat, List<Debugger.V2
 
             return result;
         }
+
+        /// <summary>
+        /// Formats the variable into a more readable string,
+        /// especially if the variable only has members and no value.
+        /// </summary>
+        /// <returns>Formatted string representing the variable.</returns>
+        private string FormatVariable(Debugger.V2.Variable variable)
+        {
+            if (variable.Status?.IsError ?? false)
+            {
+                return $"\"Error evaluating {variable.Name}: {variable.Status?.Description?.Format}\"";
```

Comment:
Don't we still need to format the status message with the params?

Reply:
I am not doing that because the debugger just puts the message directly in `Format` and does not give any `Parameters`.